src/settings/handlers/AccountSettingsHandler.ts

```diff
@@ -110,6 +110,21 @@ export default class AccountSettingsHandler extends MatrixClientBackedSettingsHa
             return content ? content['enabled'] : null;
         }
 
+        // Special case for autoplaying videos and GIFs
+        if (["autoplayGifs", "autoplayVideo"].includes(settingName)) {
+            const settings = this.getSettings() || {};
+            const value = settings[settingName];
+            // Fallback to old combined setting
+            if (value === null || value === undefined) {
+                const oldCombinedValue= settings["autoplayGifsAndVideos"];
```

Comment:
Looks like somehow it went through linter on CI.
```suggestion
                const oldCombinedValue = settings["autoplayGifsAndVideos"];
```